e2e/tests/test_e2e.py

```diff
@@ -117,6 +117,8 @@ def setUpClass(cls):
             yaml.dump(configmap, f, Dumper=yaml.Dumper)
 
         for filename in ["operator-service-account-rbac.yaml",
+                         "postgresql.crd.yaml",
```

Comment:
why configure operator twice here ? once with configmap and once with CRD?

Reply:
the operatorconfiguration is not used. This is only to test if the CRD can be created. Maybe I should add the sample manifest too.

Reply:
In that case it might be a first small e2e test of its own. The whole purpose of this `setUp` class method is to prepare the infra for actual tests, not to test anything